### PR TITLE
Create 2024-02-01-geospatial-techno.md

### PR DESCRIPTION
**Pull request by geospatialtechno**
Please review my content and let me know if any necessary corrections are needed.

### CODE REVIEW

File: _posts/2024-02-01-geospatial-techno.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+- Cascaded Services include WFS, WMS, and WMTS. GeoServer can proxy a remote Web Map Service and Web Map Tile Service and load it as a store in GeoServer.
+
+Other data sources are supplied as GeoServer extensions. Extensions are downloadable modules that add functionality to GeoServer. Click the appropriate data source to configure the store, because the connection parameters vary depending on data format.
+
+To create a Shapefile data store, follow these steps:
+- Select the desired workspace from the drop-down menu.
+- Enter the Data Source Name. Make sure the Enabled option is checked. Otherwise, access to the store along with all the datasets defined, will be disabled for it.
+- In the "Shapefile location", click on the "Browse" link to define the location of the shapefile.
+- DBF charset is used to decode strings from the dbf file, so it's very important to select an appropriate DBF character set.
```
Comment (jodygarnett) [inline _posts/2024-02-01-geospatial-techno.md:63]: I rarely have to use this option; I think it is only needed as an "override" when the shape file could not figure it out?

> DBF charset can be supplied, the character set could not be determined correctly from the DBF file

**Review by jodygarnett — CHANGES_REQUESTED** [review]
Some minor feedback provided below.

Videos are great and accessible to people that do not wish to read instructions. That makes maintaining the instructions a bother.

Q: Can you think of anyway to minimize duplication with the user manual tutorials?

My concern is for visitors of the blog - as the blog post may become out of date, while the user manual tutorial matches the latest version.

**Suggestion:** We indicate which version of GeoServer this blog post was written for.

Q: If we want to make this look more consistent you could follow the user manual writing guide conventions:
- using "click" for links, and "press" for button
- Use **Save** (GUI elements are always **bold**)
- Navigate to **Data > Workspaces** page (we always call them pages)
- Use ``text`` to indicate content in lists to select
- Etc...

Writing conventions are important in a user manual where everything needs to be consistent. Blog post it is not so important so this is only for you to consider.

**Suggestion:** Please think about writing guidlines; I think if you did the **bold** for gui-elements and `text` for list and layer name selection it would be helpful. Following the same phrases as the user manual (click and press) are probably too much - as the blog post would not sound like your content and how you are speaking in the video.

**Review by geospatialtechno — COMMENTED** [review]
@jodygarnett I have made the edits to this blog post according to your instructions.
Thanks for reviewing it

**Comment by jodygarnett** [comment]
Thanks, I will review this morning

**Comment by jodygarnett** [comment]
Thanks for waiting; I finally have my mkdocs conversion "done" (content not navigation) and can come up for air to breath.

**Pull request merged**